include/iota/crypto/checksum.hpp

```diff
@@ -26,28 +26,20 @@
 #pragma once
 
 #include <iota/crypto/kerl.hpp>
+#include <iota/crypto/sponge_factory.hpp>
 
 namespace IOTA {
 
 namespace Crypto {
 
-class Checksum {
-public:
-  Checksum();
-  virtual ~Checksum();
+namespace Checksum {
 
-public:
-  Types::Trytes add(const Types::Trytes& address);
-  Types::Trytes remove(const Types::Trytes& address) const;
-  bool         isValid(const Types::Trytes& address);
+Types::Trytes add(const Types::Trytes& address);
```

Comment:
can you add documentation for these functions? We can split it in another PR, but can be a good opportunity to improve doc coverage.

Reply:
Sure, it'll help for #101